Gulpfile.mjs

```diff
@@ -108,13 +109,14 @@ if (process.env.CIRCLE_PR_NUMBER) {
   versionSuffix = "+pr." + process.env.CIRCLE_PR_NUMBER;
 }
 
-const babelVersion =
-  require("./packages/babel-core/package.json").version + versionSuffix;
-function buildRollup(packages) {
+const babelVersionNumber = await import("./packages/babel-core/package.json")
+  .version;
```

Comment:
Since `.json` files cannot be imported, we need something like this:

```js
import { createRequire } from "module";

const require = createRequire(import.meta.url);

const babelVersionNumber = require("./packages/babel-core/package.json").version;
```